### PR TITLE
Upgrade Capital Framework components and use their brand colors instead of cf-theme-overrides

### PR DESCRIPTION
**Pull request by cfarm**
Use the latest Capital Framework components so that we no longer have to maintain brand colors in this repository.

## Removals

- Deletes `cf-theme-overrides.less` and `brand-palette.less` from the project and removes them from `main.less`

## Changes

- CFPB-branded color variables are now defined directly in the Capital Framework components that are upgraded in `package.json`
- Color updates for CF components should be managed upstream in Capital Framework's [`cf-brand-colors.less`](https://github.com/cfpb/capital-framework/blob/master/src/cf-core/src/cf-brand-colors.less) file

## Testing

1. You can test this locally, but pretty much all the colors will remain unchanged on the front end, because most variables are duplicated in this repo's atomic components anyway.
1. check out this branch
1. run `./frontend.sh`
1. `./runserver.sh`and `gulp watch` in another window
1. confirm styles are loading as expected

## Notes

-

## Todos

-

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

**Comment by Scotchester** [comment]
There are lots of non-color variables in `cf-theme-overrides.less`. Did those all get set in CF to match their cfgov-refreh values?

**Comment by cfarm** [comment]
@scranfill Yes, all the sizing and other variables got set in each component with the values from this repo's cf-theme-overrides.less.

**Comment by anselmbradford** [comment]
Pagination on /about-us/newsroom/ doesn't look right:
screen shot 2017-09-11 at 1 15 18 pm

**Comment by anselmbradford** [comment]
Re-generate the `package-lock.json`.

**Comment by cfarm** [comment]
@anselmbradford Hmm odd! here's the pagination on capital framework: https://cfpb.github.io/capital-framework/components/cf-pagination/

**Comment by cfarm** [comment]
@anselmbradford so the pagination background color is the same issue we had that kicked this whole business off to begin with: outdated colors in `cfpb-chart-builder`'s [`cf-theme-overrides.less`](https://github.com/cfpb/cfpb-chart-builder/blob/v3.0.1/src/css/cf-theme-overrides.less#L212). Until the next version of `cfpb-chart-builder` is bumped in this repo, there may be wrong colors popping up like this. I can see if a chart builder release is ready for going live, and we can ship these two PRs together if we're really worried about it.

**Comment by cfarm** [comment]
OK, the latest version of cfpb-chart-builder will fix the pagination colors, so I'm bumping it in this PR too. Release notes for that are here: https://github.com/cfpb/cfpb-chart-builder/releases

**Comment by anselmbradford** [comment]
@cfarm have you pushed up your local changes? I don't see chart builder being bumped.

**Comment by cfarm** [comment]
@anselmbradford just pushed, this is ready for review again. now that chart builder's overrides file is gone, i had to make some variable names consistent with what's in Capital Framework, so those changes are in here too.

**Comment by cfarm** [comment]
fonts issue looks to be the same one as seen in CF - https://github.com/cfpb/capital-framework/pull/591 looking at fix now.

**Comment by cfarm** [comment]
OK, so with the removal of cf-theme-overrides, the project needs to declare the font variables and import the licensed-fonts.css file from cf-typography in another file (these were done in the overrides file). I added these to main.less. I think it's worth putting those variables into Capital Framework like we did with colors, but I'll address that separately (or put in an issue for it).

Working fonts now:
screen shot 2017-09-13 at 12 21 35 pm

**Comment by anselmbradford** [comment]
Is this ready to look at again?

**Comment by cfarm** [comment]
Yep!

File: cfgov/unprocessed/css/on-demand/secondary-navigation.less [hunk 1]
```diff
@@ -24,10 +24,10 @@
 @import (reference) 'cf-typography.less';
 
 // The Capital Framework (CF) brand color palette.
-@import (reference) "../brand-palette.less";
+// @import (reference) "../brand-palette.less";
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/on-demand/secondary-navigation.less]: Can we delete this commented out bits or add a `TODO` on when they can be deleted?

Comment (cfarm) [reply cfgov/unprocessed/css/on-demand/secondary-navigation.less]: oops, meant to delete it - thanks!

**Comment by cfarm** [comment]
Deled the comments, once tests pass I'll merge.

**Pull request merged**